api/db-cdr/changelog-schema/db.changelog-8-achilles-indexes.xml

```diff
@@ -0,0 +1,93 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="biopete" id="changelog-8 add concept indexes ">
+    <validCheckSum>ANY</validCheckSum>
+
+    <createIndex
+                 indexName="idx_concept_name"
+                 schemaName="cdr"
+                 tableName="concept"
+                 unique="false">
+      <column name="concept_name" type="varchar(255)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_concept_code"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="concept_code" type="varchar(50)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_domain_id"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="domain_id" type="varchar(20)"/>
+    </createIndex>
+
+    <createIndex
+            indexName="idx_vocabulary_id"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="vocabulary_id" type="varchar(20)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_standard_concept"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="standard_concept" type="varchar(1)"/>
+    </createIndex>
+
+
+  </changeSet>
+
+  <changeSet author="biopete" id="changelog-8.1 add achilles_results_concept indexes ">
+
+    <validCheckSum>ANY</validCheckSum>
+    <createIndex
+            indexName="idx_analysis_id"
+            schemaName="cdr"
+            tableName="achilles_results_concept"
+            unique="false">
+      <column name="analysis_id" type="varchar(255)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_analysis_id_stratum_1"
+            schemaName="cdr"
+            tableName="achilles_results_concept"
+            unique="false">
+      <column name="analysis_id" type="bigint(20)"/>
+      <column name="stratum_1" type="varchar(255)"/>
+    </createIndex>
+
+    <createIndex
+            indexName="idx_analysis_id_stratum_1_2"
+            schemaName="cdr"
+            tableName="achilles_results_concept"
+            unique="false">
+      <column name="analysis_id" type="bigint(20)"/>
+      <column name="stratum_1" type="varchar(255)"/>
+      <column name="stratum_2" type="varchar(255)"/>
+    </createIndex>
+
+  </changeSet>
+
+  <changeSet author="biopete" id="changelog-8.2 add achilles_results count_value index ">
+
+    <validCheckSum>ANY</validCheckSum>
```

Comment:
Not sure this is needed? This file should be run once and validate on real checksum.

Reply:
According to dan these will all get deleted and redone when we roll out finalized db. So i'm going to wait on optimizing change logs from a final db dump.